### ISSUE TITLE
Clarify encryption Scope

### ISSUE THREAD

**Issue by J-a-k-o-b** [MicrosoftDocs/microsoft-365-docs#4753]
Hi,

Here the Product Group shared that Customer KEy now supports MSFT Teams:
https://techcommunity.microsoft.com/t5/microsoft-teams-blog/customer-key-support-for-microsoft-teams-now-in-public-preview/ba-p/1999893

In this doc https://docs.microsoft.com/en-us/microsoft-365/compliance/customer-key-overview?view=o365-worldwide#customer-key-encrypts-data-at-rest-in-office-365
is listed what is encrypted using Customer Key.

Here Teams Conversations and Chats are not listed. Please add it or make a clear delimitation from it. Now it's unclear if only Teams Files were supported and handled by Customer Key Encryption or the whole Suite.

Regards#Jakob

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a585eca1-dad6-3afd-43ce-8c4b11102dd3
* Version Independent ID: a1d708a7-774f-0c23-b428-28919ef5d46a
* Content: [Service encryption with Customer Key - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/customer-key-overview?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/customer-key-overview.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/customer-key-overview.md)
* Service: **o365-seccomp**
* GitHub Login: @KCCross
* Microsoft Alias: **krowley**

**Comment by msbemba** [comment]
@J-a-k-o-b Thank you for your feedback. We have made changes to the document via PR#https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4819

Thanks
Sri

**Comment by J-a-k-o-b** [comment]
Thank you @msbemba for updating it. So, every components of Teams supports customer key encryption?

**Comment by msbemba** [comment]
@J-a-k-o-b That is correct. More information can be found here -https://docs.microsoft.com/en-us/microsoft-365/compliance/customer-key-tenant-level?view=o365-worldwide

**Comment by yogkumgit** [comment]
@J-a-k-o-b I will close the issue, after PR is merged. Reopening.

**Comment by tiburd** [comment]
@J-a-k-o-b and @yogkumgit - [this related PR](https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4819) hasn't been merged yet, but to me, it looks like recent changes to the article might address the concerns is this issue:
- [Service encryption with Customer Key](https://docs.microsoft.com/microsoft-365/compliance/customer-key-overview)

@J-a-k-o-b - can you please review the article and let us know whether your suggestions are now incorporated? Please @ mention me in your reply. Thank you!
